projects/noloan2/app/src/main/java/hasadna/noloan2/LawsuitPdf.java

```diff
@@ -0,0 +1,248 @@
+package hasadna.noloan2;
+
+import android.Manifest;
+import android.content.pm.PackageManager;
+import android.os.Build;
+import android.os.Environment;
+import android.support.v4.app.ActivityCompat;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.util.Log;
+import android.view.View;
+import android.widget.Button;
+
+import com.itextpdf.licensekey.LicenseKey;
+import com.itextpdf.text.DocumentException;
+import com.itextpdf.html2pdf.*;
+
+import java.io.BufferedReader;
+import java.io.FileOutputStream;
+import java.io.FileReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.TimeZone;
+
+import noloan.R;
+
+public class LawsuitPdf extends AppCompatActivity { // TODO: Add logs to class functions.
```

Comment:
Let's rename to `LawsuitPdfActivity`
From now on, renaming will be written as:
`s/LawsuitPdf/LawsuitPdfActivity`
`s` is for switch.

Reply:
Done.